Articles of Governance/Voting Procedures.md

```diff
@@ -1,9 +1,36 @@
-# Voting Procedures Act
+# Legislative Procedure
+Providing the Knights with a clear and simple process for creating and voting on legislation.
 
-This Act is to provide a clear and simple process for voting on legislation.
+## Article I - Proposing Legislation
+### Section I
+When crafting some hopeful legislation, remember that the Articles of Governance are the code of **how** governing works within the April Knights. And that Legislation provides the additional context about the changes where appropriate. Where commentary is useful to the application of "the articles", that commentary should be added within the articles themselves. All other parts of an act that aren't specifically about how governing is done should go within a separate legislative act.
```

Comment:
When is legislation formally proposed? When any speaking member of LC or UC says that they formally propose a bill? Or when the GM/FR announce a proposed bill in the conclave?